aspnet/hosting/aspnet-core-module.rst

```diff
@@ -0,0 +1,155 @@
+|                           | |                                                  |
+|                           | | Specifies the relative or absolute file path for |
+|                           | | which **stdout** and **stderr** from the process |
+|                           | | specified in **processPath** will be logged.     |
+|                           | | Relative paths are relative to the root of the   |
+|                           | | site. Any path starting with '.' will be         |
+|                           | | relative to the site root and all other paths    |
+|                           | | will be treated as absolute paths.               |
+|                           | |                                                  |
+|                           | | The default value is ``httpplatform-stdout``.    |
++---------------------------+----------------------------------------------------+
+| forwardWindowsAuthToken   | | True or False.                                   |
+|                           | |                                                  |
+|                           | | If  true, the token will be forwarded to the     |
+|                           | | child process listening on %HTTP_PLATFORM_PORT%  |
+|                           | | as a header 'X-IIS-WindowsAuthToken' per request.|
+|                           | | It is the responsibility of that process to call |
+|                           | | CloseHandle on this token per request.           |
+|                           | |                                                  |
+|                           | | The default value is false.                      |
++---------------------------+----------------------------------------------------+
+
+Child Elements
+^^^^^^^^^^^^^^^
+
++---------------------------+----------------------------------------------------+
+| Attribute                 |     Description                                    |
++===========================+====================================================+
+| environmentVariables      | | Configures **environmentVariables** collection   |
+|                           | | for the process specified in **processPath**.    |
++---------------------------+----------------------------------------------------+
+
+ASP.NET Core Module app offline
+-------------------------------
+
+If you place a file with the name  *app_offline.htm* at the root of a web application directory, the ASP.NET Core Module will shut-down the application and stop processing any new incoming requests for that application. The ASP.NET Core Module will respond to all requests for dynamic pages in the application by sending back the content of the  *app_offline.htm*  file. For example, you might want to return a "site under construction" or a "down for maintenance" page.
+
+Once the *app_offline.htm* file is removed, the next request will load the application and the application will respond to requests.
```

Comment:
These clauses are a little funky at the end, since we sort of expect not to find an independent clause back there. How about ...

> Once the *app_offline.htm* file is removed, the next request will load the application, which will then respond to requests.